### ISSUE TITLE
Add aria-activedescendant regression tests that ensure the element referenced by aria-activedescendant is visible when it is changed via keyboard commands

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#2802]
PR #2622 fixed an issue in listbox example code where the code failed to satisfy the requirement that an element referenced by aria-activedescendant is made visible when aria-activedescendant is changed by pressing a keyboard command, such as arrows, home, end, pageUp, PageDown, etc.

Currently, we don't have regression tests that check to see if this requirement is met.

Devise a testing method and implement it for all examples that use aria-activedescendant.
